static/.eslintrc

```diff
@@ -133,6 +166,7 @@ Do not edit this file in your local project
     "no-useless-rename": 2,
     "no-whitespace-before-property": 2,
     "no-with": 2,
+    "object-curly-spacing": [2, "always"],
```

Comment:
Please keep it at `never`
https://github.com/upfrontIO/livingdocs/blob/master/contribution-guidelines/style-guides/coffeescript.md#our-rules